src/integrationtest/python/GeoAttribute_tests.py

```diff
@@ -0,0 +1,48 @@
+# Name: Debasish Kanhar
+
+import unittest
+from subprocess import Popen, PIPE
+from JanuaGraphContainer import JanusGraphContainer
+from janusgraph_python.driver.ClientBuilder import JanusGraphClient
+from gremlin_python.structure.graph import Graph
+from janusgraph_python.core.attribute.Geo import Geo
+from janusgraph_python.core.datatypes.GeoShape import GeoShape
+
+
+class TestGeoAttributes(unittest.TestCase):
+    def setUp(self):
+        self.container = JanusGraphContainer()
+
+        docker_ip = Popen(["docker-machine", "ip"], stdout=PIPE).communicate()[0]
```

Comment:
same comment as above, please look into using `self.container.get_container_host_ip()` instead of using Popen to get the ip.